README.md

```diff
@@ -17,3 +17,9 @@ OPI subgroup dealing with provisioning and lifecycle management.
     * [lifecycle-subgroup](https://opi-project.slack.com/archives/C0342L6T7EC)
 * Status
     * Active
+
+# I Want To Contribute!
+
+This project welcomes contributions and suggestions.  We are happy to have the Community involved via submission of **Issues and Pull Requests** (with substantive content or even just fixes). We are hoping for the documents, test framework, etc. to become a community process with active engagement.  PRs can be reviewed by by any number of people, and a maintainer may accept.
+
+See [CONTRIBUTING](../opi/CONTRIBUTING.md) and [GitHub Basic Process](../opi/doc-github-rules.md) for more details.
```

Comment:
Does this link work across repos?

Reply:
nope

Reply:
OK, you would need to wait until https://github.com/opiproject/opi/pull/3 merges and then put the full link in.

Reply:
yep